environment.py

```diff
@@ -331,6 +331,11 @@ def env() -> Mapping[str, Optional[str]]:
         #
         'AZUL_GOOGLE_SERVICE_ACCOUNT_PUBLIC': 'azul-ucsc-{AZUL_DEPLOYMENT_INCARNATION}-public-{AZUL_DEPLOYMENT_STAGE}',
 
+        # The name of the Google Cloud service account to be created and used
+        # to simulate access from users who are logged in but not registered
+        # with SAM.
+        'AZUL_GOOGLE_SERVICE_ACCOUNT_UNREGISTERED': 'azul-ucsc-{AZUL_DEPLOYMENT_INCARNATION}-unreg-{AZUL_DEPLOYMENT_STAGE}',
```

Comment:
```suggestion
        'AZUL_GOOGLE_SERVICE_ACCOUNT_UNREGISTERED': 'azul-ucsc-{AZUL_DEPLOYMENT_INCARNATION}-unregistered-{AZUL_DEPLOYMENT_STAGE}',
```

Unless that breaks a limit on the length.

Reply:
```
 git diff
diff --git a/environment.py b/environment.py
index c5476921..ccd56ee1 100644
--- a/environment.py
+++ b/environment.py
@@ -311,7 +311,7 @@ def env() -> Mapping[str, Optional[str]]:
         # The name of the Google Cloud service account to be created and used
         # to simulate access from users who are logged in but not registered
         # with SAM.
-        'AZUL_GOOGLE_SERVICE_ACCOUNT_UNREGISTERED': 'azul-ucsc-{AZUL_DEPLOYMENT_INCARNATION}-unreg-{AZUL_DEPLOYMENT_STAGE}',
+        'AZUL_GOOGLE_SERVICE_ACCOUNT_UNREGISTERED': 'azul-ucsc-{AZUL_DEPLOYMENT_INCARNATION}-unregistered-{AZUL_DEPLOYMENT_STAGE}',
 
         # The number of concurrently running lambda executions for the
         # contribution and aggregation stages of indexing, respectively.
$ make deploy
...
terraform validate

Error: "account_id" ("azul-ucsc-6-unregistered-nadove") doesn't match regexp "^[a-z](?:[-a-z0-9]{4,28}[a-z0-9])$"

  on authentication.tf.json line 66, in resource[0].google_service_account[2].azul_unregistered:
  66:                     }


make[1]: *** [Makefile:25: validate] Error 1
make[1]: Leaving directory '/home/nadove/PycharmProjects/azul-sc/terraform'
make: *** [Makefile:81: deploy] Error 2
```